internal/serve/serve.go

```diff
@@ -283,16 +283,7 @@ func handler(deps handlerDeps) http.Handler {
 			r.Post("/create-fee-bump", accountHandler.CreateFeeBumpTransaction)
 		})
 
-		r.Route("/transactions", func(r chi.Router) {
-			handler := &httphandler.TransactionsHandler{
```

Comment:
I don't see the `TransactionHandler` used elsewhere in this PR, can we remove it? Or is it used by another REST endpoint?